From 7ea14c3170a11eb1a9ec35163092941962bc01d8 Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Mon, 30 Jul 2012 16:44:20 -0700
Subject: Make full_x86 mk files to be similar to full

This fixes full_x86 build, as full_x86 mk files are out-of-date.

Change-Id: I3cdca8de49656e27e3ee80ca5f8cb40838e4fabd
Depends-Change-Id: I14f3041b92c3d22c22d8756f57f8d5ab556516d0
Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 target/board/generic_x86/device.mk |    8 +++++++-
 target/product/full_x86.mk         |    2 ++
 2 files changed, 9 insertions(+), 1 deletions(-)

diff --git a/target/board/generic_x86/device.mk b/target/board/generic_x86/device.mk
index 574ce4f..40da03c 100644
--- a/target/board/generic_x86/device.mk
+++ b/target/board/generic_x86/device.mk
@@ -25,7 +25,13 @@ PRODUCT_PROPERTY_OVERRIDES := \
 PRODUCT_COPY_FILES := \
     device/generic/goldfish/data/etc/apns-conf.xml:system/etc/apns-conf.xml \
     device/generic/goldfish/camera/media_profiles.xml:system/etc/media_profiles.xml \
+    $(call add-to-product-copy-files-if-exists,development/tools/emulator/system/camera/media_profiles.xml:system/etc/media_profiles.xml) \
+    $(call add-to-product-copy-files-if-exists,development/tools/emulator/system/camera/media_codecs.xml:system/etc/media_codecs.xml) \
+    hardware/libhardware_legacy/audio/audio_policy.conf:system/etc/audio_policy.conf \
     device/generic/goldfish/camera/media_codecs.xml:system/etc/media_codecs.xml
 
 PRODUCT_PACKAGES := \
-    audio.primary.goldfish
+    audio.primary.goldfish \
+    power.goldfish \
+
+# end of file
diff --git a/target/product/full_x86.mk b/target/product/full_x86.mk
index 4dff713..c23262b 100644
--- a/target/product/full_x86.mk
+++ b/target/product/full_x86.mk
@@ -32,6 +32,8 @@ ifdef NET_ETH0_STARTONBOOT
   PRODUCT_PROPERTY_OVERRIDES += net.eth0.startonboot=1
 endif
 
+$(call inherit-product, $(SRC_TARGET_DIR)/product/full_base_telephony.mk)
+
 # Ensure we package the BIOS files too.
 PRODUCT_PACKAGES += \
 	bios.bin \
-- 
1.7.4.1

